packages/syntax-core/src/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,155 @@
+import React, { forwardRef, type ReactNode, type ReactElement } from "react";
+import { type Placement as RAPlacement } from "react-aria";
+import {
+  Tooltip as RACTooltip,
+  TooltipTrigger as RACTooltipTrigger,
+} from "react-aria-components";
+
+import Triggerable from "../react-aria-utils/Triggerable";
+import Typography from "../Typography/Typography";
+import OverlayArrow from "../react-aria-utils/OverlayArrow";
+import Box from "../Box/Box";
+import OverlayVisibility from "../react-aria-utils/OverlayVisibility";
+
+type Placement = "top-end" | "top-start" | "bottom-end" | "bottom-start";
+
+const SYNTAX_PLACEMENT_TO_RAC_PLACEMENT: Record<Placement, RAPlacement> = {
+  "top-end": "top right",
+  "top-start": "top left",
+  "bottom-end": "bottom right",
+  "bottom-start": "bottom left",
+} as const;
```

Comment:
Block talk to design but no actual code changes needed.

Would be great if Popover and Tooltip can share the `placement`. Or for `Tooltip` we can just to `top` and `bottom`

Reply:
https://github.com/Cambly/syntax/pull/303/commits/938df8af43fa20a5324adf07d497bd69fe4f76cb

Tooltip restricted to `top` and `bottom` only.

Reply:
followup:

https://github.com/Cambly/syntax/pull/303/commits/5c196dcdf98e59a6a5f6d2c3a53ea04496f6ea3b

Tooltip and Popover props consolidated.  options are `top` `bottom` `start` `end`